src/main/java/Robots/samples/ObstacleAvoidRobot.java

```diff
@@ -54,6 +55,33 @@ public void loop() throws Exception {
             } else {
                 motion.move(defaultMoveSpeed, defaultMoveSpeed, 1000);
             }
+        }*/
+
+        if(state == robotState.RUN){
+            double dist = distSensor.getDistance();
+            System.out.println("dist  3 " +distSensor.getDistance());
+
+
+            if(dist<=10){
```

Comment:
It is much better if you can taken out this value, `10` as a global/class variable